### DISCUSSION TITLE
Is there a way to have prisma generate a seed script for a database?

### DISCUSSION THREAD

**Discussion by HartS** [prisma/prisma#6023] [Q&A] [answered]
For a new project, I want the database to be seeded with some initial data for dev and testing in the team.

I've created the data in a sqlite db through Prisma Studio. If I want to share this with my teammates, do I need to send them the .dev.db file directly? Or manually write the seed file?

**Comment by ryands17** [comment, marked as answer]
@HartS 👋 

Prisma cannot automatically create a seed script. You would need to create one and populate data as described [here](https://www.prisma.io/docs/guides/prisma-guides/seed-database). If you have created the data via Prisma studio, you can share the  `dev.db` database, but it would be much easier to share the seed file and run the seed commands.

Let me know if you have any other queries.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
